app/controllers/accounts_controller.rb

```diff
@@ -30,13 +29,10 @@ def destroy
 
   private
 
-  def account_params
-    params.require(:account)
-          .permit(:subdomain,
-                  owner_attributes: [
-                    :first_name, :last_name, :email,
-                    :password, :password_confirmation]
-                 )
+  def signup_params
+    params.require(:signup)
+          .permit(:subdomain, :first_name, :last_name, :email,
+                    :password, :password_confirmation)
```

Comment:
Hmm, we actually can't right now since rails throws almost everything from our request into the `params` hash. We could move the whitelisting into the forms though, by doing something like this in the constructor:

```
def initialize(params)
  super(sanitize params)
  build_children
end

private

def sanitize(params)
  params.require(:signup).permit(:subdomain, :first_name, :last_name, :email, :password, :password_confirmation)
end
```